Template structure (#41)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/blueprint/dropin-features.mdx

---
title: DROPIN_NAME features
description: Learn about the purpose and features of the DROPIN_NAME dropin.
---

import Diagram from '@components/Diagram.astro';
import Callouts from '@components/Callouts.astro';
import Aside from '@components/Aside.astro';

> 

Reviewer: ```suggestion
import OptionsTable from '@components/OptionsTable.astro';

```

{/* *_ 1. Answer the question: What is this dropin designed to do? 2. Describe use cases that are supported and what is not supported._* */}

The DROPIN_NAME dropin provides...

## Features

> The DROPIN_NAME dropin provides the following features for your storefront:

Reviewer: ```suggestion
The following table provides an overview of the Adobe Commerce features that the DROPIN_NAME dropin supports:

<OptionsTable
  options={[
    ['Feature', 'Status'],
    ['Name of Commerce feature', 'Supported, API only, Roadmap'],
    ['Name of Commerce feature', 'Supported, API only, Roadmap'],
  ]}
/>
```

> {/**_ This diagram should show a screenshot with all the main elements shown with content.

Reviewer: I think that we should reserve the "Features" H2 for the dropin + Adobe Commerce feature matrix. Then, maybe have another H2 called "Anatomy" or "Examples" where we show a diagram of the elements on a storefront page similar to waht we're [already showing for PDP](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-details/). In the existing PDP docs, I'm not sure if the "Anatomy" and "Examples" sections are supposed to be different or are actually the same thing.

Author: Yeah, I like that. I'll ensure the topic of the PDP Overview and the dropin template features are synced. Thanks!

> {/**_ This diagram should show a screenshot with all the main elements shown with content.

Reviewer: ```suggestion
## Anatomy

{/**_ This diagram should show a screenshot with all the main elements shown with content.
```

---

import Diagram from '@components/Diagram.astro';
import Callouts from '@components/Callouts.astro';
import Aside from '@components/Aside.astro';
import OptionsTable from '@components/OptionsTable.astro';

{/* *_ 1. Answer the question: What is this dropin designed to do? 2. Describe use cases that are supported and what is not supported._* */}

The DROPIN_NAME dropin provides...

## Features

The following table provides an overview of the Adobe Commerce features that the DROPIN_NAME dropin supports:

<OptionsTable
  options={[
    ['Feature', 'Status'],
    ['Name of Commerce feature', 'Supported, API only, Roadmap'],
    ['Name of Commerce feature', 'Supported, API only, Roadmap'],
  ]}
/>

## Anatomy

{/**_ This diagram should show a screenshot with all the main elements shown with content.
Circle numbers should be added beside each element so they can be described in the Callouts component below. _**/}

<Diagram caption="[DropinElements]">![PDP slots](@images/placeholder.webp)</Diagram>

<Callouts>

1. **Element1**: [AddDescription]
1. **Element2**: [AddDescription]
1. **Element3**: [AddDescription]
1. **Element4**: [AddDescription]

</Callouts>

## Customizable elements

> The DROPIN_NAME dropin currently provides the following elements, most of which are customizable:

Reviewer 2: ```suggestion
The DROPIN_NAME dropin provides the following elements, most of which are customizable:
```
